Jenkinsfile

```diff
@@ -32,4 +32,18 @@ fhBuildNode(['label': 'nodejs']) {
             distCmd = 'default'
         }
     }
+
+    stage('Build Image') {
+        final String version = getBaseVersionFromPackageJson()
+        final String tag = "${version}-${env.BUILD_NUMBER}"
+        final Map params = [
+                fromDir: './docker',
+                buildConfigName: 'fh-template-apps',
+                imageRepoSecret: 'dockerhub',
+                outputImage: "docker.io/rhmap/feedhenry-sdks:${tag}"
```

Comment:
Would produce an image here docker.io/rhmap/feedhenry-sdks:5.0.13-747. This didn't previously have a version only the build number since the image contained more than just template apps, but this is more in keeping with all the rest of our components.

Repo name should be changed from 'feedhenry-sdks' to ideally 'fh-template-apps', would require changes to templates.